DOCSP-41216 - Broken link (#29)
mongodb/docs-cpp: source/configuration.txt

in 3.6, and zstd compression in 4.2.

> Data compression can be enabled with the appropriate URI options, as documented

Reviewer: ```suggestion
You can enable data compression the appropriate URI options. Learn about these options
```

Author: yeah, left the text alone because this page is currently getting redone as part of standardization. can fix though

> in the `API documentation for the MongoDB C driver <https://mongoc.org/libmongoc/current/mongoc_uri_t.html#connection-options>`__.

Reviewer: ```suggestion
in the `MongoDB C driver API documentation <https://mongoc.org/libmongoc/current/mongoc_uri_t.html#connection-options>`__.
```
